anchore_engine/services/policy_engine/api/controllers/feeds.py

```diff
@@ -215,6 +214,11 @@ def delete_feed(feed):
 @authorizer.requires_account(with_types=INTERNAL_SERVICE_ALLOWED)
 def delete_group(feed, group):
     session = db.get_session()
+    if feed == "grypedb":
```

Comment:
See my other comment about having a single source of truth for the name of the grype db feed that we can reference wherever needed

Reply:
This now uses the `GRYPE_DB_FEED_NAME` constant from sync_utils.py